kafka-plugins-common/pom.xml

```diff
@@ -15,7 +15,7 @@
   <dependencies>
     <dependency>
       <groupId>io.cdap.cdap</groupId>
-      <artifactId>cdap-data-pipeline2_2.11</artifactId>
+      <artifactId>cdap-data-pipeline3_2.12</artifactId>
       <version>${cdap.version}</version>
```

Comment:
What is the current cdap.version? It's a bigger change, we may want to bump it

Reply:
CDAP version is `6.8.0-SNAPSHOT` in the develop branch.

I've updated the CDAP API range from `[6.7.0-SNAPSHOT:7.0.0-SNAPSHOT)`